src/dbnode/storage/namespace/types.go

```diff
@@ -104,6 +109,15 @@ type IndexOptions interface {
 	BlockSize() time.Duration
 }
 
+// Schema represents namespace schema
+type Schema interface {
+	Version() uint32
```

Comment:
nit: mind adding comments for all methods

Reply:
sure